.gitignore

```diff
@@ -8,3 +8,6 @@ node_modules/
 # Local Netlify folder
 .netlify
 .env
+
+# Webstorm config files
+.idea
```

Comment:
[nit] IDE config files shouldn't be in our gitignore file. Could you please remove?